### DISCUSSION TITLE
Color token overlay - Button Icon

### DISCUSSION THREAD

**Discussion by henriquebarrosr** [Telefonica/mistica-design#1821] [Q&A] [answered]
Hoy, dentro de la documentación del componente Button Icon, tenemos una parte que explica la aplicación de un overlay de un token de color sobre su color principal, para aplicar los efectos de Hover y Pressed.

![Image](https://github.com/Telefonica/mistica-design/assets/161517581/41633ffe-1a0b-4fbf-ab16-033c21fe5724)

Sin embargo, al explorar la construcción del componente, nos dimos cuenta de que también hay una variación en la opacidad de este token que se aplica.

![imagem (1)](https://github.com/Telefonica/mistica-design/assets/161517581/9713f848-49e9-4642-8a60-6e0192a2448f)
![imagem](https://github.com/Telefonica/mistica-design/assets/161517581/7a483107-4eba-4e02-8c45-9306179da389)

¿Crees que no tendría sentido explicar esta diferencia dentro de la documentación? Entendemos que esto podría ayudar tanto a los diseñadores como los desarrolladores a comprender un poco mejor este funcionamiento.

¡Gracias!

**Comment by aweell** [comment]
Hola @henriquebarros-telef 

Totalmente, mi sugerencia aquí es la siguiente:

* Creamos una issue para actualizar esa parte
* Actualizamos las specs con una explicación ampliada de cómo se comportan esos estados
* Ponemos a "Design Ops Vivo" como revisor de la rama, para que confirméis que este detalle se entiende mejor

**Comment by henriquebarrosr** [comment]
Hola @aweell, cómo estás?
Sí, genial. Prefieres que abramos una issue para esto aquí o lo haces por ahí?

**Reply by aweell** [reply]
Hello! @henriquebarros-telef,

Creemos que una manera de mejorar como presentamos la información es mostrarlo de la siguiente manera:

![Screenshot 2024-09-03 at 13 06 07](https://github.com/user-attachments/assets/431eb560-44c9-405e-a592-f73583b5dbb3)

Si esto ayuda, actualizamos directamente sin falta de issue, si hay algún punto mas que necesite revisión podrías indicarnos que puntos son criticos para entender la documentación?

Gracias

**Reply by henriquebarrosr** [reply]
Hola, @aweell! Que tal?
De mi parte, genial! Podremos hacer así. Solo que este punto era realmente más crítico de entender (que había un token de opacidad superpuesto al de la marca, para crear un efecto de _hover_).
Gracias por todo!

**Reply by aweell** [reply, marked as answer]
Perfecto, queda actualizado así en las specs:

https://www.figma.com/design/koROdh3HpEPG2O8jG52Emh/%F0%9F%94%B8-Buttons-Component-Specs?m=auto&node-id=3752-1610&t=8CaBBb9AC4YfddgF-1

Cierro la discussion.
